content/id/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,1062 @@
+---
+title: Secrets
+content_template: templates/concept
+feature:
+  title: Secret dan manajemen konfigurasi
+  description: >
+    Menerapkan serta mengubah secret serta konfigurasu aplikasi tanpa melakukan perubahan pada image kamu serta mencegah tereksposnya secret yang kamu miliki pada konfigurasi.
+weight: 50
+---
+
+
+{{% capture overview %}}
+
+Objek `secret` pada Kubernetes mengizinkan kamu menyimpan dan mengatur informasi yang sifatnya sensitif, seperti
+_passwords_, token OAuth, dan ssh _keys_. Menyimpan informasi yang sifatnya sensitif ini ke dalam `secret`
+cenderung lebih aman dan fleksible jika dibandingkan dengan menyimpan informasi tersebut secara apa adanya pada
+definisi {{< glossary_tooltip term_id="pod" >}} atau di dalam {{< glossary_tooltip text="container image" term_id="image" >}}.
```

Comment:
```suggestion
definisi {{< glossary_tooltip term_id="pod" >}} atau di dalam {{< glossary_tooltip text="kontainer image" term_id="image" >}}.
```

(based on other localized uses of “kontainer” vs. English “container” in this page)